aries-site/src/layouts/main/Layout.js

```diff
@@ -72,7 +72,7 @@ export const Layout = ({ children, title }) => {
                   />
                 </Box>
               )}
-              <Box flex>{mainContent[0]}</Box>
+              <Main flex>{mainContent[0]}</Main>
```

Comment:
Main is usually flex={false} why do we need to override it?

Reply:
The Main is supposed to fill the rest of the width of the window that the sidebar isn't taking up. Flex allows it to expand to fill this width. When I remove it, the section divider ( hr ) doesn't span to the far right.